### ISSUE TITLE
Seems FontThumbnailService isn't fully working on Azure AppService

### ISSUE THREAD

**Issue by jonascarlbaum** [Geta/Epi.FontThumbnail#17]
Same code works differently locally and in cloud _(Azure AppService)_. Some thumbs won't load in Azure, everyone works locally. Same ttf-file locally and on Azure.

Used like this, won't work on Azure:
```
[ThumbnailIcon("fa-light-300.ttf", 0xf015)]
```

Used like this, works on Azure:
```
[ThumbnailIcon("fa-light-300.ttf", 0xf1ea)]
```

Error:
```
Parameter is not valid. at System.Drawing.Bitmap..ctor(String filename)
```
![image](https://user-images.githubusercontent.com/392041/106587539-08874480-654a-11eb-8f80-b794681d6100.png)

Seems like some icons works to generate thumbnails from, and some won't.

![image](https://user-images.githubusercontent.com/392041/106587166-91ea4700-6549-11eb-8a38-612005dd5b2e.png)
Some files seems to be zero sized too.

---

It seems like `new Bitmap(path)` won't work reliably in Cloud see [here ](https://stackoverflow.com/a/14747906) and [here](https://docs.microsoft.com/en-us/dotnet/api/system.drawing?redirectedfrom=MSDN&view=dotnet-plat-ext-5.0) since .

Error exists in [FontThumbnailService.cs](https://github.com/Geta/Epi.FontThumbnail/blob/master/src/Geta.Epi.FontThumbnail/FontThumbnailService.cs).

Proposed fix might be changing this, if `new Bitmap` works best with `stream` instead of `path`:
```
if (File.Exists(cachePath))
{
    using (var bmpTemp = new Bitmap(cachePath))
    {
       return new Bitmap(bmpTemp);
    }
}
```

to this _(dummy-coded)_:
```
if (File.Exists(cachePath))
{
    using (StreamReader sr = new StreamReader(cachePath)) 
    {
        using (var bmpTemp = new Bitmap(sr))
        {
            return new Bitmap(bmpTemp);
        }
    }
}
```

**Comment by jonascarlbaum** [comment]
Clearing cached thumbnails didn't work.
But I managed to solve it by clearing the cached thumbnails and restart Azure AppService.

So that's an interesting behaviour. ;)

**Comment by degborta** [comment]
Yeah, that's strange.. I'll give it a look and see if I can reproduce it.

**Comment by jonascarlbaum** [comment]
@degborta Seems like same behaviour occurs over time on local machine too — can check error logs the next time it happens. It happened after adding new thumbnails and Re-compiled. IIS-reset after clearing thumbs works locally too as a ”fix”.
